cmd/koord-manager/main.go

```diff
@@ -186,3 +195,14 @@ func setRestConfig(c *rest.Config) {
 		c.Burst = *restConfigBurst
 	}
 }
+
+func installHTTPHandler(mgr ctrl.Manager) error {
```

Comment:
does this need to move into the extensions/options directory for internal modifications?

Reply:
not necessary since metrics.InternalRegistry/ExternalRegistry are global